config/install/core.entity_form_display.node.stanford_news.default.yml

```diff
@@ -5,13 +5,16 @@ dependencies:
     - field.field.node.stanford_news.body
     - field.field.node.stanford_news.field_s_news_category
     - field.field.node.stanford_news.field_s_news_date
-    - field.field.node.stanford_news.field_s_news_hero_banner
+    - field.field.node.stanford_news.field_s_news_feat_image
```

Comment:
they didnt have any content on production so I already added the field manually on production to keep them unblocked.